scripts/github-changelog.php

```diff
@@ -178,7 +180,7 @@ function create_changelog_for_last_PR() {
         exit( 0 );
     }
 
-    $changelog_record = parse_changelog_html( $changelog_html );
+    $changelog_record = parse_changelog_html( $changelog_html, WP_TITLE_PREFIX );
 
     create_draft_changelog( $changelog_record['title'], $changelog_record['content'], $changelog_tags );
```

Comment:
I think we should set the prefix here rather than passing it to `parse_changelog_html`.